workflows/subclustering/🍒_RelevanceAI_Subclustering.ipynb

```diff
@@ -0,0 +1,17241 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    n_clusters = 10
</pre><p>All hard-coded variables to the top please so users can just fill out a form and run </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/RelevanceAI/workflows/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Do you want me to make this a Colab form input?</p>